varia/style-rtl.css

```diff
@@ -3368,6 +3368,10 @@ img#wpstats {
  * Full Site Editing
  * - Full Site Editing overrides
  */
+.fse-enabled .site-footer {
+	display: block;
+}
+
```

Comment:
This change appears to be a leftover from https://github.com/Automattic/themes/pull/1379 (see [#](https://github.com/Automattic/themes/pull/1379/files#diff-b6d805de0184172207b2b53df97b66d7)).
I think what happened is that my editor's Git extension for some reasons didn't refresh the modified files after running `npm run build`, and this file here wasn't picked up, and I didn't realize because it's automatically compiled.
Now after checking out the new branch, here it is this unexpected change.
It's a oneliner in a pretty simple PR, so I'm inclined to leave it here regardless, instead of creating a new PR just for that.